scripts/upload_test_results/upload_to_github.py

```diff
@@ -0,0 +1,175 @@
+#!/usr/bin/python3
+import glob
+import os
+import argparse
+import shutil
+from git import Actor, Repo, Git
+import json
+import sys
+import logging
+import configparser
+import urllib.request
+from github import Github
+
+logging.basicConfig(format='%(levelname)s:%(message)s', level=logging.INFO)
+
+def find_ver_json(commit_name):
+    # Download versions.json and find commit name
+    ver_url = 'https://testing.zephyrproject.org/daily_tests/versions.json'
+
+    try:
+        webURL = urllib.request.urlopen(ver_url)
+        data = webURL.read()
+        ver_data = json.loads(data)
+
+        for ver_name in reversed(ver_data):
+            if ver_name == commit_name:
+                logging.info('Found existing version %s', ver_name)
+                return ver_name
+    except Exception:
+        logging.critical("ERROR: Can't download versions.json or find given version in that file. Script execution ended.")
+        sys.exit(1)
+
+def parse_args():
+    # Read arguments from the command line
+    parser = argparse.ArgumentParser(description = "Script to upload Daily Test Results .xml files to the test_results repo on Github")
```

Comment:
Your code style is inconsistent, for example the spaces around `=` is unnecessary.
```
description = "Script to upload Daily Test Results .xml files to the test_results repo on Github"
```
 You can use `flake8` or other tools to check the code style.

Reply:
Good advice, I forgot about that. Thx.